src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -52,7 +47,8 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
   private var inProgressChat = CancellationToken()
   private val stopGeneratingButton =
       JButton("Stop generating", IconUtil.desaturate(AllIcons.Actions.Suspend))
-  private val recipesPanel: JBPanelWithEmptyText
+  private val commandsPanel: CommandsTabPanel =
+      CommandsTabPanel(project) { cmd: String -> sendMessage(project, message = null, cmd) }
```

Comment:
I wonder... does it make sense to have a type for commands?

Reply:
It does and I will add it, it was just a quick WiP PR to check if it works.